docs/reference/objects/product/variant_price.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Variant Price
+parent: Product
+---
+
+This object represents a price of a variant. When using a `price` object you have access to the following attributes:
+
+# price.fractional
+
+The amount for this occupancy represented in the subunit of the current user's currency.
+e.g. considering the amount $19.50, it will return 1950.
+
+# price.per_person_fractional
+
+The amount per-person represented in the subunit of the current user's currency.
+e.g. considering the amount $19.50, it will return 1950
+
+# price.occupancy
+
+The occupancy associated with this price, i.e. if the variant is purchased with this occupancy, this is what the price will be.
+
+# price.price
+
+The amount for this occupancy represented in the subunit of the current user's currency.
+e.g. considering the amount $19.50, it will return 1950.
+
+> **Deprecated**
```

Comment:
[Question] as before should we put this where we're first mentioning the method to be really explicit